tests/test_schema_rules.py

```diff
@@ -28,9 +28,15 @@ def test_lowercase_ids():
             ), "The id in {} should be lower case".format(schema["id"])
 
 
-def test_id_length():
+def test_nodeid_length():
+    prefix_len = len("node_")
+    postfix_len = len("_node_id_idx")
```

Comment:
`_node_id_idx` is not the longest index name - see link in the ticket comments

once fixed, please comment the links here to explain where these pre/postfix strings come from, as well as why 63 🙂